recipes/prettytable/meta.yaml

```diff
@@ -0,0 +1,23 @@
+package:
+  name: prettytable
+  version: "7"
```

Comment:
To explain this a little bit. Conda has so called channels - like PPA on Ubuntu. You can mix channels and enable multiple ones. But Conda has a concept of channel priority so you need to enable the channels in a correct order. Every bioconda package has also conda-forge enable. So you can mix bioconda-packages with conda-forge ones.